src/config.rs

```diff
@@ -880,6 +880,8 @@ pub struct DbConfig {
     pub max_total_wal_size: ReadableSize,
     pub max_background_jobs: i32,
     #[config(skip)]
+    pub max_background_flushes: i32,
```

Comment:
`pub max_background_flushes: Option<i32>`

Reply:
No, I think the missing case is covered by `default()`.